README.md

```diff
@@ -29,6 +29,9 @@ class Post < ActiveRecord::Base
     column { Author.arel_table[:name] }
     column { Arel::Nodes::NamedFunction.new('CONCAT', [arel_table[:prefix], arel_table[:suffix]]) }
 
+    # Allow custom wildcard replacement using a match e.g. searching for `"My*Post"` will query `ILIKE 'My%Post'`.
```

Comment:
pls also explain that searching for `My Post` without quotes will result in `ILIKE 'My%Post%'` prefix query.